### PR TITLE
Add repr html evoked

### PR DESCRIPTION
**Pull request by AJQuinn**
#### Reference issue
closes https://github.com/mne-tools/mne-python/issues/9177
supersedes and thus closes https://github.com/mne-tools/mne-python/pull/9220

#### What does this implement/fix?
This takes the _repr_html method from https://github.com/mne-tools/mne-python/pull/9220 and updates it to use the neew jinja2 templates.

### CODE REVIEW

File: doc/changes/latest.inc [hunk 1]
```diff
@@ -31,6 +31,8 @@ Enhancements
 - :meth:`mne.Epochs.plot_psd_topomap` now suppresses redundant colorbars when ``vlim='joint'`` (:gh:`11051` by `Daniel McCloy`_)
 - Add ``starting_affine`` keyword argument to :func:`mne.transforms.compute_volume_registration` to initialize an alignment with an affine (:gh:`11020` by `Alex Rockhill`_)
 - The ``trans`` parameter in :func:`mne.make_field_map` now accepts a :class:`~pathlib.Path` object, and uses standardised loading logic (:gh:`10784` by :newcontrib:`Andrew Quinn`)
+- Add HTML representation for `~mne.Epochs` and `~mne.Evoked` in Jupyter Notebooks (:gh:`9220` by `Valerii Chirkov`_)
```
Comment (drammock) [inline doc/changes/latest.inc:34]: this will need to be updated. add your name after Valerii

**Review by drammock — COMMENTED** [review]
please make a trivial change to `tutorials/evoked/10_evoked_overview.py`. Editing a tutorial will cause CircleCI to execute that tutorial so we can see what the `_repr_html_` actually looks like, before merging.

File: doc/changes/latest.inc [hunk 2]
```diff
@@ -31,6 +31,8 @@ Enhancements
 - :meth:`mne.Epochs.plot_psd_topomap` now suppresses redundant colorbars when ``vlim='joint'`` (:gh:`11051` by `Daniel McCloy`_)
 - Add ``starting_affine`` keyword argument to :func:`mne.transforms.compute_volume_registration` to initialize an alignment with an affine (:gh:`11020` by `Alex Rockhill`_)
 - The ``trans`` parameter in :func:`mne.make_field_map` now accepts a :class:`~pathlib.Path` object, and uses standardised loading logic (:gh:`10784` by :newcontrib:`Andrew Quinn`)
+- Add HTML representation for `~mne.Epochs` and `~mne.Evoked` in Jupyter Notebooks (:gh:`9220` by `Valerii Chirkov`_ and `Andrew Quinn`_)
```
Comment (drammock) [inline doc/changes/latest.inc:34]:
```suggestion
- Add HTML representation for `~mne.Epochs` and `~mne.Evoked` in Jupyter Notebooks (:gh:`11075` by `Valerii Chirkov`_ and `Andrew Quinn`_)
```

**Review by drammock — COMMENTED** [review]
please change line 261 of the tutorial from `print(left_right_aud)` to simply `left_right_aud` (this will trigger the `_repr_html_`, I forgot that explicitly calling `print` will override it).

Note also that I edited the PR description to explicitly use the word `closes` so that the issue and the other PR will automatically close when this one gets merged.

**Comment by AJQuinn** [comment]
Thanks @drammock - that makes sense. Hopefully fixed in the last commit.

File: doc/changes/latest.inc [hunk 3]
```diff
@@ -31,8 +31,8 @@ Enhancements
 - :meth:`mne.Epochs.plot_psd_topomap` now suppresses redundant colorbars when ``vlim='joint'`` (:gh:`11051` by `Daniel McCloy`_)
 - Add ``starting_affine`` keyword argument to :func:`mne.transforms.compute_volume_registration` to initialize an alignment with an affine (:gh:`11020` by `Alex Rockhill`_)
 - The ``trans`` parameter in :func:`mne.make_field_map` now accepts a :class:`~pathlib.Path` object, and uses standardised loading logic (:gh:`10784` by :newcontrib:`Andrew Quinn`)
-- Add HTML representation for `~mne.Epochs` and `~mne.Evoked` in Jupyter Notebooks (:gh:`9220` by `Valerii Chirkov`_ and `Andrew Quinn`_)
-
+- Add HTML representation for `~mne.Epochs` and `~mne.Evoked` in Jupyter Notebooks (:gh:`11075` by `Valerii Chirkov`_ and `Andrew Quinn`_)
+- Allow :func:`mne.beamformer.make_dics` to take ``pick_ori='vector'`` to compute vector source estimates (:gh:`19080` by `Alex Rockhill`_)
```
Comment (drammock) [inline doc/changes/latest.inc:35]:
```suggestion
```

**Review by drammock — APPROVED** [review]

File: doc/changes/latest.inc [hunk 4]
```diff
@@ -31,6 +31,7 @@ Enhancements
 - :meth:`mne.Epochs.plot_psd_topomap` now suppresses redundant colorbars when ``vlim='joint'`` (:gh:`11051` by `Daniel McCloy`_)
 - Add ``starting_affine`` keyword argument to :func:`mne.transforms.compute_volume_registration` to initialize an alignment with an affine (:gh:`11020` by `Alex Rockhill`_)
 - The ``trans`` parameter in :func:`mne.make_field_map` now accepts a :class:`~pathlib.Path` object, and uses standardised loading logic (:gh:`10784` by :newcontrib:`Andrew Quinn`)
+- Add HTML representation for `~mne.Epochs` and `~mne.Evoked` in Jupyter Notebooks (:gh:`11075` by `Valerii Chirkov`_ and `Andrew Quinn`_)
```
Comment (agramfort) [inline doc/changes/latest.inc:34]:
```suggestion
- Add HTML representation for `~mne.Evoked` in Jupyter Notebooks (:gh:`11075` by `Valerii Chirkov`_ and `Andrew Quinn`_)
```

**Pull request merged**